tests/bundle/Core/EventListener/BackwardCompatibleCommandListenerTest.php

```diff
@@ -46,7 +46,7 @@ public function testGetSubscribedEvents(): void
             [
                 ConsoleEvents::COMMAND => [['onConsoleCommand', 128]],
             ],
-            $this->listener->getSubscribedEvents()
+            $this->listener::getSubscribedEvents()
```

Comment:
I'm surprised this is valid syntax.

:flushed: 

:+1:

Reply:
> I'm surprised this is valid syntax.

:sunglasses: